requirements.txt

```diff
@@ -1,4 +1,5 @@
 prusa.connect.sdk.printer>=0.7.1dev0
+gcode_metadata
```

Comment:
Why? SDK depends on gcode_metadata yet

Reply:
Oh, I did pip install, it  succeeded, but gcode_metadata was still missing. We import it here, so let's have it in requirements here, so let's have it in the requirements here
![image](https://github.com/prusa3d/Prusa-Link/assets/5272796/d6ee9195-32ea-4f72-b64c-4336ff2a75c0)